components/typography/styles.js

```diff
@@ -2,7 +2,7 @@ import '../colors/colors.js';
 import { css, unsafeCSS } from 'lit';
 
 export const _isValidCssSelector = (selector) => {
-	const re = /[#.]?([a-zA-Z0-9-_]+)(\[[a-zA-Z0-9-_]+\])?([a-zA-Z0-9-_]+)?/g;
+	const re = /([a-zA-Z0-9-_ >.#]+)(\[[a-zA-Z0-9-_]+\])?([a-zA-Z0-9-_ >.#]+)?/g;
```

Comment:
Rough idea:
- 0 or more alphanumeric (plus `-`, `_`, ` `, `>`, `.`, `#`
- optional `[ ... ]` (inside brackets can be alphanumeric plus `-` and `_`)
- 0 or more alphanumeric (plus `-`, `_`, ` `, `>`, `.`, `#`